http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/CorsSupport.scala

```diff
@@ -23,20 +27,49 @@ class CorsSupport(contentSecurityPolicy: ContentSecurityPolicyConfig) {
     )
   }
 
+  /** Whether to allow any Origin header value or to restrict to the allowlist.
+   * Enable strict mode by either removing the wildcard (*) from
```

Comment:
This feels redundant, aka the flag is set to True only in environments where there is a wildcard (BEE and alpha), so why have it?

Reply:
Setting this to true when there is a wildcard in the list lets Origins be checked against a strict allowlist while any Referer is permitted.